ui/widgets/nuxeo-selectivity.js

```diff
@@ -7373,6 +7400,38 @@ typedArrayTags[weakMapTag] = false;
         this._triggerQueryCallback(query, results);
       });
     }
+
+    _getScrollParent() {
+      let scrollParent = document.body;  // use the document.body as the default scrollable parent
+      let style = getComputedStyle(this);
+      const excludeStaticParent = style.position === 'absolute';
+      const overflowRegex = /(auto|scroll)/;
+
+      if (style.position !== 'fixed') {
+        let parent = this;
+        while (parent) {
+          if (parent.parentElement) {
+            parent = parent.parentElement;
+          } else if (parent.getRootNode()) {
+            parent = parent.getRootNode();
+            if (parent) {
```

Comment:
Isn't the last `if` going to evaluate as `true` every time? You already checked for `parent` in the previous `if`, when it was still `parent.getRootNode()`. 
The way I see it:
```
} else if (a) {
  b = a;
  if (b) {
```
If you checked for `a` before, why check for `b` when `b` is `a`?

Reply:
You're right Miguel. This is what happens when you start refactoring code and don't clean up stuff properly. Thanks, I'll update that.